icons/bruno.svg

```diff
@@ -0,0 +1 @@
3047.26572-.99371 1.08102-4.23494.02343-6.58618a.47224.47224 0 0 0-.62502-.23634.47224.47224 0 0 0-.23634.62502c.83488 1.8561.17267 5.0301-.07422 5.95334-.0788.29474-.04727.6066.08203.87894l.23048.4844c-.12602.24312-.37402.69026-.72854 1.17777-.5265.724-1.27978 1.48947-2.0821 1.75202a.47224.47224 0 0 0-.30275.59572.47224.47224 0 0 0 .59572.3008c1.12836-.36924 1.96998-1.28967 2.55478-2.09383.17746-.24403.32846-.47806.45705-.68948l.08203.17188c.39566.83333 1.5646.98396 2.15438.2715.65298-.7889 1.5245-1.96234 2.15242-3.24817s1.04193-2.70724.60744-3.99038c-1.3101-3.86905-2.89027-5.699-4.20913-6.54907-.65943-.42504-1.24884-.60004-1.6817-.66214-.31902-.04576-.44002-.017-.55276-.0039-1.96954-1.26835-3.86986-1.72598-5.50215-1.7364M8.7126 8.35846c-.27591-.00027-.54525.1402-.69924.39259-.23466.38459-.1119.8869.27344 1.12113.3853.23423 1.12114-.27344 1.12114-.27344s.11384-.88691-.2715-1.12114a.816.816 0 0 0-.42384-.11914m7.4202 0a.816.816 0 0 0-.42385.11914c-.38533.23423-.2715 1.12114-.2715 1.12114s.7358.50767 1.12114.27344.5081-.73654.27345-1.12113c-.154-.25241-.42333-.39287-.69925-.3926m-5.25215 3.5802c-.36515 0-.6956.2284-.82425.57034l-.44533 1.18364a.4723.4723 0 0 0 .26173.60353l.94535.38869c.50885.20903 1.10318-.11645 1.20121-.65823a.47224.47224 0 0 0-.38087-.54885.47224.47224 0 0 0-.50588.31447l-.47658-.19532.26759-.71292h2.42587l.28517.66995-.4551.24024a.47224.47224 0 0 0-.48634-.33204.47224.47224 0 0 0-.40627.52932c.07858.59233.75259.9415 1.2813.66408l.89066-.46681a.4723.4723 0 0 0 .21485-.60354l-.47268-1.11137c-.13786-.32382-.45895-.53517-.81057-.53517zm-2.9962 2.3829a.47224.47224 0 0 0-.52932.40627c-.11387.862.16681 1.5957.584 2.13094.4025.51637.91797.86772 1.36334 1.10941-.01426.35174-.02217.98448.06055 1.66022.06203.50682.17287 1.0385.39259 1.51177.21971.47327.5765.91454 1.12309 1.08207.5948.18232 1.17248.24375 1.67193.2129.49946-.03085.92271-.1262 1.25005-.40626.35342-.30238.58003-.78495.77932-1.52936.17343-.64776.27388-1.472.18165-2.42.06101-.02458.12084-.05274.1797-.08399.45605-.24219.99496-.60152 1.41216-1.13676.41719-.53524.69787-1.26893.584-2.13093a.47224.47224 0 0 0-.53127-.40627.47224.47224 0 0 0-.40626.52931c.08014.60671-.0956 1.04481-.3926 1.42584-.29698.38102-.72797.68222-1.10941.8848-.08185.04346-.16683.07385-.25392.0918a.47.47 0 0 0-.07617 0 .47.47 0 0 0-.05664.01757c-.21947.01325-.43864-.05212-.61916-.19336l-.87308-.68362V15.388a.47224.47224 0 0 0-.47267-.47072.47224.47224 0 0 0-.47268.47072v1.0059l-.87307.68362c-.28879.22597-.67842.25789-1.0059.08398-.38145-.20256-.81439-.50377-1.11138-.8848s-.47273-.81912-.39259-1.42583a.47224.47224 0 0 0-.40626-.52932m4.26187 2.90245.7637.59768c.27045.2116.5871.33613.9141.37697.07618.81899-.01281 1.52094-.15626 2.05671-.16668.6226-.4765 1.05133-.48049 1.05473-.0455.03893-.32005.16042-.69533.1836-.37529.02318-.85007-.02489-1.33599-.17383-.21435-.0657-.39167-.24802-.54494-.57815s-.25575-.7808-.31056-1.22856c-.05997-.48999-.06685-.96368-.06055-1.29692.40613-.001.8095-.13388 1.14262-.39455z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Bruno</title><path d="M9.394 9.583s-.742.511-1.13.275c-.387-.236-.51-.742-.274-1.129.236-.388.742-.511 1.129-.275.388.236.275 1.129.275 1.129m1.48 3.245-.308.82.582.24c.04-.224.254-.372.477-.332.224.041.372.254.332.478-.093.515-.638.813-1.122.613l-.95-.391a.41.41 0 0 1-.228-.524l.448-1.193c.12-.32.426-.533.769-.533h2.525l.239.036c.076.023.148.057.214.101a.817.817 0 0 1 .303.363l.476 1.12c.083.197.002.425-.187.524l-.896.471c-.505.266-1.121-.053-1.196-.62a.411.411 0 0 1 .353-.461.411.411 0 0 1 .461.353l.56-.294-.327-.771zM9.353 17.98c-.959-.514-2.183-1.54-1.961-3.22.029-.225.236-.383.461-.354.225.03.383.236.353.461-.167 1.272.822 2.009 1.546 2.395.348.185.771.151 1.081-.092l.903-.708v-1.045c0-.226.184-.41.411-.41s.411.184.411.41v1.045l.903.708c.19.15.423.22.655.209a.45.45 0 0 1 .074-.019.42.42 0 0 1 .093-.003c.089-.02.176-.051.259-.095.724-.386 1.713-1.123 1.546-2.395a.411.411 0 0 1 .353-.461c.225-.029.432.129.461.354.223 1.688-1.013 2.716-1.974 3.227-.072.038-.145.071-.22.099.148 1.424-.14 2.562-.449 3.24-.169.37-.355.617-.49.732-.194.167-.517.305-.933.366-.531.078-1.227.039-1.945-.182-.576-.177-.973-.686-1.208-1.343-.362-1.009-.352-2.368-.33-2.919m4.54.213a1.81 1.81 0 0 1-.939-.376l-.807-.633-.807.633c-.341.268-.758.397-1.173.384-.012.453-.001 1.219.15 1.922.072.333.174.652.329.908.122.2.275.361.486.425.586.18 1.152.218 1.585.155.227-.033.412-.086.518-.176.115-.099.259-.363.395-.744.213-.597.367-1.46.263-2.498m-8.683-.955-.191.402c-.384.81-1.475.955-2.053.27-.81-.961-1.948-2.505-2.548-4.068-.427-1.115-.576-2.241-.213-3.198C2.894 3.559 6.247 2.969 6.98 2.928c1.34-.865 5.583-2.807 10.424.342.795-.051 4.042.203 6.424 7.252.325.961.162 2.081-.263 3.186-.599 1.557-1.709 3.088-2.5 4.046-.574.695-1.675.553-2.061-.262l-.133-.28c-.219.369-.524.832-.901 1.281-.57.679-1.306 1.32-2.154 1.598a.412.412 0 0 1-.519-.263.4114.4114 0 0 1 .263-.518c.705-.231 1.306-.78 1.781-1.345.556-.662.938-1.355 1.097-1.666l-.245-.518c-.126-.264-.154-.56-.078-.842.246-.924.981-4.015.071-6.043a.4105.4105 0 1 1 .749-.336c.993 2.212.243 5.583-.026 6.591a.405.405 0 0 0 .026.278l.811 1.712c.129.27.494.32.685.089.748-.904 1.801-2.346 2.367-3.817.351-.912.52-1.834.252-2.628-2.447-7.241-5.667-6.687-5.667-6.687a.416.416 0 0 1-.309-.061C12.48.972 8.456 2.917 7.339 3.676a.41.41 0 0 1-.252.07S3.739 3.649.973 10.935c-.297.783-.137 1.701.213 2.612.565 1.475 1.644 2.927 2.408 3.833.192.228.555.177.683-.092l.816-1.723a.4.4 0 0 0 .027-.278c-.269-1.008-1.02-4.38-.027-6.592a.4113.4113 0 0 1 .543-.206c.207.093.3.336.207.543-.911 2.028-.176 5.119.071 6.043.075.282.047.578-.078.842l-.203.427c.117.226 1.249 2.351 2.958 2.911.216.071.334.303.263.519a.4115.4115 0 0 1-.519.262c-1.484-.487-2.606-1.983-3.125-2.798m10.247-7.655s-.114-.893.274-1.129a.821.821 0 0 1 1.129.275c.236.387.113.893-.274 1.129-.388.236-1.129-.275-1.129-.275"/></svg>
```

Reply:
@adamrusted Thank you for the path. I just checked it and it looks good to me. I just perceive the dog's right eye is slightly bigger than the left eye. Should I apply this path instead of @uncenter's?

Reply:
Yeah, give it a go - then @uncenter can review it again ;)

Reply:
Alright, just updated the SVG.